templates/terraform/examples/firestore_document_nested_document.tf.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_firestore_document" "<%= ctx[:primary_resource_id] %>" {
+  collection = "somenewcollection"
+  document_id = "my-doc-%{random_suffix}"
+  fields     = "{\"something\":{\"mapValue\":{\"fields\":{\"akey\":{\"stringValue\":\"avalue\"}}}}}"
+}
+
+resource "google_firestore_document" "sub_document" {
```

Comment:
Can you add an extra level of nesting? Either here or in another test.

Reply:
Sure !